tests/test_average_pooling_1d.py

```diff
@@ -1,99 +0,0 @@
-import numpy as np
```

Comment:
I wasn't sure if I should incorporate the adjoint tests into the new test file, or leave them here but refactoring them so they pass. I removed these temporarily so the build can run on the other things. If we think these are useful to test, we can probably refactor them into the new test file easily.

Reply:
I'm OK with removing these tests.  This change contributes to #185, too.